README.md

```diff
@@ -1,2 +1,2 @@
-# Public Key Based Encryption
-Drupal 8 Module
+# Pubkey Encrypt
+Drupal 8 Module for adding Password-based Public Key Encryption support. The encryption mechanism is based on ownCloud's Data Encryption Model.
```

Comment:
This plugin system would basically handle the algorithm for retrieving/generating the public key logins